data/fields/seamark/beacon_isolated_danger/shape.json

```diff
@@ -1,5 +1,41 @@
 {
     "key": "seamark:beacon_isolated_danger:shape",
     "type": "combo",
-    "label": "Shape"
+    "label": "Shape",
+    "strings": {
+        "options": {
+            "pile": {
+                "title": "Pile Beacon",
+                "description": "A long, heavy pile or section of metal, wood or concrete, driven into the seabed, used as a navigational aid."
+            },
+            "stake": {
+                "title": "Stake",
+                "description": "An elongated wood or metal stake, embedded in the bottom of the sea, used as a navigational aid."
+            },
+            "pole": {
+                "title": "Pole",
+                "description": "An elongated wood or metal pole, embedded in the bottom of the sea, used as a navigational aid."
+            },
+            "withy": {
```

Comment:
https://wiki.openstreetmap.org/wiki/Tag:seamark:beacon_isolated_danger:shape=withy is not existing

https://taginfo.openstreetmap.org/tags/seamark:beacon_isolated_danger:shape=withy shows zero uses

https://taghistory.raifer.tech/#***/seamark%3Abeacon_isolated_danger%3Ashape/withy shows that it was never ever used

have I messed up links, value is wrong here or is it adding support to value that never was used even once?